lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -110,6 +110,22 @@ def cmd_info_help
             print_line
           end
 
+          def print_module(mod, dump_json: false, show_doc: false)
```

Comment:
Think I'd prefer this to be called something like `print_info` thoughts?

Reply:
I wasn't sure what to call it as I'm not used to naming conventions yet, so If you think `print_info` would be better I can change that.

Reply:
`print_module_info`

🥇